libnetwork/iptables/iptables_test.go

```diff
@@ -141,12 +147,16 @@ func TestLink(t *testing.T) {
 }
 
 func TestPrerouting(t *testing.T) {
+	iptable, natChain, _, err := createNewChain()
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Perhaps we can make `createNewChain()` a `t.Helper()` then we can skip all the "per test" error handling.

Reply:
Thanks @thaJeztah , that looks a lot cleaner in the code!